### DISCUSSION TITLE
Quarto-Pub consistently throws 'Page not Found', even with a newly created quarto project

### DISCUSSION THREAD

**Discussion by Hezel2000** [quarto-dev/quarto-cli#4386] [Q&A]
I have one project that I published with quarto-pub with no problem, all very nice. I now wanted to do the same for another project, with a different account, but only get 'Page not Found' errors. I started from scratch with a new project, did not change anything, and directly tried to publish the project I created. Again 'Page not Found'. However the preview on my Quarto-Pub account site looks fine. Any ideas?

**Comment by softwarenerd** [comment]
Hi @Hezel2000. Could you share the project (i.e. on GitHub) and the command / console output you're seeing for the project so I can try to reproduce your issue?

Thanks!

Brian

**Comment by Hezel2000** [comment]
Thanks Brian – here is the GitHub Link
https://github.com/Hezel2000/1GCWebsite
And here is what the terminal says:
<img width="557" alt="image" src="https://user-images.githubusercontent.com/86461739/219498821-473cdfb7-4374-49b0-bacb-33937be1ca04.png">
In the preview it looks normal:
<img width="351" alt="image" src="https://user-images.githubusercontent.com/86461739/219498967-79a10921-0dd6-4c01-83ec-bd0975e243c0.png">
(I stripped it to its bare bones)
But the site itself does not work:
https://onegeochemistry-qpub.quarto.pub/1gcwebsite/

**Comment by Hezel2000** [comment]
I tested this again (and again). It has to do with the second account. If I publish a website with one account – everything works perfect. If I switch to the other account, that the quarto-pub command offers me at the prompt – I get the page not found. It is entirely opaque to me, who is going wrong here.
